slowmovie.py

```diff
@@ -90,10 +90,42 @@ def video_info(file):
     frameTime = 1000 / fps
 
     return {
-        'frame_count' : frameCount,
-        'fps' : fps,
-        'duration' : duration,
-        'frame_time' : frameTime }
+        "frame_count" : frameCount,
+        "fps" : fps,
+        "duration" : duration,
+        "frame_time" : frameTime }
+
+# Calculate how long it'll take to play a video.
+# output value: "d[ay]", "h[our]", "m[inute]", "s[econd]", "all"; omit for an automatic guess
+def estimate_runtime(delay, increment, videoLengthS, videoFPS, output="guess"):
+    # Recurse to generate all estimates in one string
+    if output == "all":
+        return f"{estimate_runtime(delay, increment, videoLengthS, videoFPS, 's')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'm')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'h')} / {estimate_runtime(delay, increment, videoLengthS, videoFPS, 'd')}"
+
+    # Calculate runtime lengths in different units
+    frames = videoLengthS*videoFPS
```

Comment:
Why not pass this `videoInfo['frame_count']` instead?

Reply:
I'm not sure 😅  That's way simpler!

Reply:
Changed: a2baaf5